Dockerfile

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARG PG_MAJOR_VERSION=9.6
+ARG PG_MINOR_VERSION=9.6
```

Comment:
how are these used? why we need them separately? isn't it like 9 - major, 6 - minor?

Reply:
[PostgreSQL version policy](https://www.postgresql.org/support/versioning/) changed after 9.6, so 9.6 is also major version.

In order to match distribution version with docker base image, we have to separate the version.

Reply:
I have updated the default `PG_MINOR_VERSION` and `BASE_IMAGE_VERSION` to an accurate version. This should make it more easy to understand the purpose of those versions.

Reply:
`PG_MAJOR_VERSION` is part of package, for example: `postgresql-server-dev-10` and `postgresql-server-dev-9.6`.
`PG_MINOR_VERSION` is specified version of package, for example: `postgresql-server-dev-10=10.13-1\*` and `postgresql-server-dev-10=9.6.18\*`.
`BASE_IMAGE_VERSION` is base image version.

Reply:
I have removed `PG_MAJOR_VERSION` & `PG_MINOR_VERSION` variables and I parse them from installed packages.
Only `BASE_IMAGE_VERSION` variable is remained and it is overridable.

build example:
`docker build --pull --build-arg BASE_IMAGE_VERSION=10.14 -t datasketch-postgres:10.14 .`